inspirehep/search/mappings/v5/records/jobs.json

```diff
@@ -8,57 +8,100 @@
       "numeric_detection": false,
       "properties": {
         "$schema": {
-          "type": "string"
+          "type": "keyword"
         },
         "_collections": {
-          "analyzer": "lowercase_analyzer",
-          "type": "string"
+          "normalizer": "lowercase_analyzer",
+          "type": "keyword"
         },
         "_private_notes": {
           "properties": {
             "source": {
-              "type": "string"
+              "type": "keyword"
             },
             "value": {
-              "type": "string"
+              "type": "text"
+            }
+          },
+          "type": "object"
+        },
+        "accelerator_experiments": {
+          "properties": {
+            "accelerator": {
+              "type": "text"
+            },
+            "curated_relation": {
+              "type": "boolean"
+            },
+            "experiment": {
+              "type": "text"
+            },
+            "institution": {
+              "type": "text"
+            },
+            "legacy_name": {
+              "type": "text"
+            },
+            "recid": {
+              "type": "integer"
+            },
+            "record": {
+              "properties": {
+                "$ref": {
+                  "type": "keyword"
+                }
+              },
+              "type": "object"
+            }
+          },
+          "type": "object"
+        },
+        "acquisition_source": {
+          "properties": {
+            "$ref": {
+              "type": "keyword"
```

Comment:
I think you got confused between `$ref` to reference other schema elements and `$ref` inside the fields to reference other records.